src/chart/consolidateData.js

```diff
@@ -9,6 +9,19 @@ export default function consolidateData(raw) {
 
     this.setDefaults();
 
+    //apply filters from associated controls objects
+    this.filtered_data = raw;
```

Comment:
I'm still unsure why we're applying the filters in more than one place.  Seems we can just pass this array to transformData() and avoid applying the filters for each mark.

Reply:
The domains are (mostly) being defined in transformData(), so it needs access to the raw data. Further, to define the domains when `firstfilter` is applied, transformData() actually loops through all the filter options and defines the domain that way. It's definitely possible to change that work flow, but I think it's probably best to wait until we do a more thorough refactor to really mess with it.

Reply:
Makes sense.  Although if `firstfilter` is just ignoring `filtered_data` anyway seems we can just pass it `raw_data`.